fe/fe-core/src/main/java/com/starrocks/privilege/DefaultAuthorizationProvider.java

```diff
@@ -97,6 +98,11 @@ public class DefaultAuthorizationProvider implements AuthorizationProvider {
                     .put(ObjectType.GLOBAL_FUNCTION, ImmutableList.of(
                             PrivilegeType.USAGE,
                             PrivilegeType.DROP))
+
+                    .put(ObjectType.STORAGE_VOLUME, ImmutableList.of(
+                            PrivilegeType.DROP,
+                            PrivilegeType.ALTER,
+                            PrivilegeType.USAGE))
```

Comment:
which operation should check usage on storage volume?

Reply:
Create db/table using a storage volume